src/raiden_libs/matrix.py

```diff
@@ -178,7 +179,7 @@ def _handle_message(self, room, event) -> bool:
             try:
                 message_dict = json.loads(line)
                 message = message_from_dict(message_dict)
-            except (UnicodeDecodeError, json.JSONDecodeError) as ex:
+            except (UnicodeDecodeError, json.JSONDecodeError, KeyError) as ex:
                 logger.warning(
                     "Can't parse message data JSON",
```

Comment:
The error message is a bit misleading in case of a `KeyError`. Still somewhat ok, though.

Reply:
True, I'll add another case for this.

Reply:
Moved it to the next except block which fits better.